modules/storages/app/components/storages/admin/storage_list_component.sass

```diff
@@ -0,0 +1,27 @@
+.op-storage-list
+  display: grid
+  grid-template-columns: fit-content(180px) 1fr 180px 100px 240px
+  grid-template-areas: "name label user provider time" "host host user provider time"
+
+  .op-principal--name
+    @extend .text-light
```

Comment:
Colors should be set via the system_arguments of a component because it is less error-prone and we will notice when something changes on the Primer side

Reply:
AFAICT, the `User::AvatarComponent` is not a primer component- we're still reusing the legacy avatar helper 🤔 

https://github.com/opf/openproject/blob/dev/app/components/users/avatar_component.rb